sdk/storage/Azure.Storage.Blobs/perf/Azure.Storage.Blobs.Perf/Scenarios/DownloadSasUriHttpClient.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs.Perf.Infrastructure;
+using Azure.Test.Perf;
+using CommandLine;
+
+namespace Azure.Storage.Blobs.Perf
+{
+    public class DownloadSasUriHttpClient : DownloadSasUriTest<DownloadSasUriHttpClient.BufferOptions>
+    {
+        private static readonly HttpClient _httpClient = new HttpClient();
+
+        public DownloadSasUriHttpClient(BufferOptions options) : base(options)
+        {
+        }
+
+        public override void Run(CancellationToken cancellationToken)
+        {
+            throw new NotImplementedException();
+        }
+
+        public override async Task RunAsync(CancellationToken cancellationToken)
+        {
+            using var response = await _httpClient.GetAsync(
+                SasUri,
+                Options.Buffer ? HttpCompletionOption.ResponseContentRead : HttpCompletionOption.ResponseHeadersRead,
+                cancellationToken
+                );
+
+#pragma warning disable CA2016 // Forward the 'CancellationToken' parameter to methods that take one
+            await response.Content.CopyToAsync(Stream.Null);
```

Comment:
Why suppress instead of passing the parameter?

Reply:
@pakrym: I believe the API which takes a CancellationToken only exists in .NET 5.0:

https://docs.microsoft.com/en-us/dotnet/api/system.net.http.httpcontent.copytoasync?view=net-5.0#System_Net_Http_HttpContent_CopyToAsync_System_IO_Stream_System_Threading_CancellationToken_

If there is an API which does take a CancellationToken in `netcoreapp2.1` and `net461` I can use it instead.